tests/suites/0_stateless/13_tpch/13_0005_q5.result

```diff
@@ -1,5 +1,5 @@
 CHINA	7822103.0
```

Comment:
Seems the value of CHINA sometimes is `7822102.999`, so if using `floor`, the result is 7822102.0

Reply:
-CHINA	7822103.0
+CHINA	7822104.0

I'd like to ignore the last number...

Reply:
Strange, ceil(7822103.0) -> 7822103.0, why occur 7822104.0 🤔

Reply:
sometimes it'll be 7822103.x.